Makefile

```diff
@@ -410,7 +410,7 @@ endif
 	$(INSTALL_F) $(JULIAHOME)/contrib/julia.appdata.xml $(DESTDIR)$(datarootdir)/metainfo/
 	# Install terminal info database
 ifneq ($(WITH_TERMINFO),0)
-	cp -R -L $(build_datarootdir)/terminfo $(DESTDIR)$(datarootdir)
+	cp -R -L $(build_datarootdir)/terminfo $(DESTDIR)$(datarootdir)/julia
```

Comment:
I think this and the other places might need trailing slashes? Also looks like this might need to be `mkdir -p`'d.

Reply:
I don't think the slash is needed, but adding it doesn't hurt so I've pushed a commit (also fixing a mistake).

No need to call `mkdir -p` as the dir is already created above when installing other things (Base, stdlibs...).